capsicum-test-main.cc

```diff
@@ -27,6 +30,51 @@ class SetupEnvironment : public ::testing::Environment
     }
     std::cerr << tmpdir << std::endl;
   }
+  void CheckCapsicumSupport() {
+    // For versions of googletest that lack GTEST_SKIP.
+#ifndef GTEST_SKIP
+#define GTEST_SKIP GTEST_FAIL
+#define GTEST_SKIP_defined
+#endif
+
+#ifdef __FreeBSD__
+    size_t trap_enotcap_enabled_len;
+    int rc;
+    bool trap_enotcap_enabled;
+
+    trap_enotcap_enabled_len = sizeof(trap_enotcap_enabled);
+
+    if (feature_present("security_capabilities") == 0) {
+      // XXX (ngie): using std::cerr because 1.8.1 (with GTEST_SKIP support)
+      //             isn't properly outputting skip diagnostic message here.
```

Comment:
You should include a link to the gtest issue so future maintainer will know when they can switch back to diagnostic messages.

Reply:
Filing an upstream issue for a feature we backported that doesn’t occur on master is a bit of a hard sell.. while I agree in theory, in practice, I don’t think it does much good :/.

Reply:
You mean that the problem is already fixed on gtest's master?  Ok, no need for an issue then.  How about just saying that the problem applies to gtest "< 1.8.2" or something like that?

Reply:
As I noted, after doing some investigation as part of https://reviews.freebsd.org/D19765, the backport doesn't actually work, like it does on master -_-.

So, remove the comments and unnecessary `std::cerr` calls. I'll have to track down what other commits are missing to make skipping from a test environment SetUp fixture work.